### ISSUE TITLE
`copy-frameworks` does not work for UI Testing Bundle

### ISSUE THREAD

**Issue by cybertk** [Carthage/Carthage#748]
When integrating libs with Carthage for a UI Testing Target, `/usr/local/bin/carthage copy-frameworks` does not work as expected. The UI Testing Target still cannot load framework copied by Carthage.

It seems that we need copy the frameworks to a different location (**XXXX-Runner.app**) for UI Testing Bundle.

It works fine if I copied frameworks to **XXXX-Runner.app** the `Frameworks` dir with

```
cp -r $SRCROOT/Carthage/Build/iOS/Nimble.framework $TARGET_BUILD_DIR/../Frameworks
```

**Comment by ikesyo** [comment]
Sorry for the late response, but I'm not sure why you were trying to use `copy-frameworks` for testing bundle. I think you don't need to do that. Instead, just using normal “Copy Files” phase should work as in README:

> In rare cases, you may want to also copy each dependency into the build product (e.g., to embed dependencies within the outer framework, or make sure dependencies are present in a test bundle). To do this, create a new “Copy Files” build phase with the “Frameworks” destination, then add the framework reference there as well.

**Comment by brentleyjones** [comment]
I just encountered this. While true that we can just use copy files, I like to be able to be consistent. I feel that we should change `BUILD_DIR` to `TARGET_BUILD_DIR` in copy-frameworks. Most of the time they are equal, but in this case they are not. Though, after typing this, I wonder if that will break extensions?

**Comment by brentleyjones** [comment]
And for why it's useful to use the same script: if you have a "universal target" it's hard to get the right framework with the "Copy Files" phase. We use a user-defined setting to target the correct Carthage/Build folder and framework, and use that as the source input.

**Comment by mdiep** [comment]
Fixed by #1214.
